packages/python-packages/api-stub-generator/ci.yml

```diff
@@ -38,8 +38,12 @@ extends:
         python $(Build.SourcesDirectory)/eng/scripts/python_version_check.py
       displayName: 'Verify Python APIView version consistency'
 
+    # - script: |
+    #     pip install requests
+    #     python $(Build.SourcesDirectory)/eng/scripts/check_pylintrc.py
+    #   displayName: 'Verify local pylintrc in sync with Azure SDK for Python'
```

Comment:
Should this still be here?

Reply:
Yeah, I'll probably get rid of it. Since we are only looking for our custom pylint checks, it becomes a lot less important to make sure the rcfiles are in sync. In fact, we may not need the rcfile at all.

Reply:
Removed, though this has the (potentially desirable?) effect of showing all of our custom pylint checks, including the ones that our SDK's rcfile disables.

Reply:
https://apiviewstaging.azurewebsites.net/Assemblies/Review/c15dc2842fc7461191d71959838451be